Qpf/Macro/Data/RecForm.lean

```diff
@@ -0,0 +1,66 @@
+import Qpf.Macro.Data.Replace
```

Comment:
Should this maybe be a separate PR?

Reply:
I agree. It seems like this functionality was just lifted from another file ("Ind.lean"). I think it would be good to make a small pr of the form "[NFC] Chore: refactor Recursive form functionality into a separate file".